engines/python/src/main/java/ai/djl/python/engine/Connection.java

```diff
@@ -120,10 +124,59 @@ CompletableFuture<Output> send(Input input) throws InterruptedException {
         return f;
     }
 
-    static String[] getPythonStartCmd(PyEnv pyEnv, Model model, int workerId, int port) {
+    static String[] getPythonStartCmd(
+            PyEnv pyEnv, Model model, int workerId, int port, int clusterSize, String[] hosts) {
         Device device = model.getNDManager().getDevice();
         int deviceId = device.getDeviceId();
         int tensorParallelDegree = pyEnv.getTensorParallelDegree();
+        // int pipelineParallelDegree = pyEnv.getPipelineParallelDegree();
+
+        if (clusterSize > 1) {
```

Comment:
I think it's better for us to wrap the settings into different function for better debug-bility

Reply:
Added a `PyEnv.isMultiNode()` utility function.